templates/common/_base/units/disable-mglru.service.yaml

```diff
@@ -0,0 +1,14 @@
+name: disable-mglru.service
+enabled: true
+contents: |
+  [Unit]
+  Description=Disables MGLRU on Openshfit
+  
+  [Service]
+  Type=oneshot
+  RemainAfterExit=yes
+  ExecStart=bash -c "echo 0 > /sys/kernel/mm/lru_gen/enabled"
+
+  
+  [Install]
+  WantedBy=multi-user.target
```

Comment:
Before=kubelet.service or something, since that's the thing we want to protect from starting things under MG LRU?

Reply:
Nevermind, I see below you're adding it as a dept to kubelet.